### ISSUE TITLE
Unable to attach attachment or image from attach icon using key board when screen reader is active.

### ISSUE THREAD

**Issue by Wajeed-msft** [MicrosoftDocs/msteams-docs#4731]
Copied bug from: https://github.com/microsoft/AdaptiveCards/issues/6784 

**Details from original post:**
### Target Application

Microsoft Teams

### Application Operating System

Windows

### Schema Version

NA

### Problem Description

Unable to attach attachment or image from attach icon using key board when screen reader is active. After selecting any file and clicking on enter or selecting open button and clicking enter nothing is attaching in chat bot text box.

### Expected Outcome

when screen reader is enable Teams should able to take selected attachment via chat. 

### Actual Outcome

when screen reader is enabled Teams doesn't take attachment via chat.

### Card JSON

```JSON
NA
```

### Repro Steps

_No response_---
#### Document Details

⚠ *Do not edit this section.It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Cards reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @surbhigupta12
* Microsoft Alias: **surbhigupta12**

**Comment by Nivedipa-MSFT** [comment]
@prajwalmanju - Could you please share the details of screen reader which you are using.It help us to repro the issue.

**Comment by Nivedipa-MSFT** [comment]
@prajwalmanju - We are not able to repro the issue. We are able to attach attachment or image from attach icon using key board when screen reader is active. Could you please try to update the Teams Version and check again.

**Comment by Nivedipa-MSFT** [comment]
@prajwalmanju - We are not able to repro your issue. We have tried using narrator and able to attach attachment from attach icon using key board when screen reader is active.

**Comment by prajwalmanju** [comment]
Hello Team,

We are facing this issue in JAWS screen reader tool, we are still able to reproduce this issue.

Thanks!

**Comment by Prasad-MSFT** [comment]
@prajwalmanju -We have raised a bug for this issue and concerned team is looking into it. We will inform you once we get any update. Thanks!

**Comment by Prasad-MSFT** [comment]
@prajwalmanju - Could you please share screen recording/video showing the repro steps?

**Comment by prajwalmanju** [comment]
Hello Team,

Please find the attachment for sample video showing the repro steps.

Thanks!

https://user-images.githubusercontent.com/93922557/146118588-643b8b1b-ba8b-4412-a829-6486363a9fd9.mp4

**Comment by Nivedipa-MSFT** [comment]
@prajwalmanju - Thanks for sharing the sample video. We have already raised the bug for this issue. We will update you.

**Comment by v-chetsh** [comment]
@prajwalmanju - Could you please confirm if your issue has resolved or still looking for any help?

**Comment by v-chetsh** [comment]
@prajwalmanju  - This issue has resolved now, could you please check it once from your end.
